docs/control-plane-guide/cli.mdx

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: CLI
+title: Sending a command via the CLI
+description: Sending a control plane command via the Aserto CLI
+---
+
+# Sending a command via the CLI
+
+The `aserto` CLI can be used to send commands to Edge Authorizer instances.
+
+## Required information
+
+Once logged in, the `aserto` CLI will already have the Tenant ID and authorization information to communicate with the `relay` endpoint. You will need to determine the registered instance ID. You can either do this via the [console](/docs/control-plane-guide/api#instance-id), or via the CLI.
```

Comment:
If a user has multiple tenants they need to set CLI context to the tenant they want to use.

Reply:
out of scope for this topic